### ISSUE TITLE
MLM returns false positives on idle

### ISSUE THREAD

**Issue by TaxoRubio** [runelite/runelite#1377]
The top left corner mining plugin works great, but when mining in Motherlode Mine it shows I'm idle between animations but while the vein is not depleted yet.

**Comment by Unmoon** [comment]
If you mean the idle notifier notifies you as being idle, just increase the timeout on it.

**Comment by TaxoRubio** [comment]
I mean the top left corner box with the green text that says "You are mining" and the red text that says "You are NOT mining". The red text triggers while mining the same vein.

**Comment by deathbeam** [comment]
I don't think this is solveable without making the you are mining effectively useless, because we would need to add delay to when the overlay considers when you are mining, but then the overlay will lose it's point and that is accurately determining if you are currently mining, so at that point it should be just removed and let idle notifier do that job.

**Comment by Nightfirecat** [comment]
It wouldn't be hard for us to add a short delay to resetting the mining status specifically for these wall-type mines (MLM, amethyst, daeyalt ore) and it would reduce confusion around this topic--eg. "Hey, I'm mining but the plugin says I'm not" type of questions.
